Sources/FingerprintJS/Harvesters/LocalAuthenticationInfoHarvester/LocalAuthenticationInfoHarvester.swift

```diff
@@ -0,0 +1,44 @@
+#if canImport(LocalAuthentication)
+import LocalAuthentication
+#endif
+
+@available(tvOS, unavailable)
+protocol LocalAuthenticationInfoHarvesting {
+    var isPasscodeEnabled: Bool { get }
+    var isBiometricsEnabled: Bool { get }
+    var biometryType: BiometryType { get }
+}
+
+@available(tvOS, unavailable)
+struct LocalAuthenticationInfoHarvester {
+
+    private let authenticationContextInfoProvider: AuthenticationContextInfoProviding
+
+    init(authenticationContextInfoProvider: AuthenticationContextInfoProviding) {
```

Comment:
Don't we get this for free?

Reply:
Not in this case, since the `authenticationContextInfoProvider` field is private and the compiler will only synthesize the private initializer. Yet we still need an internal initializer for writing unit tests.